docs/controls/DataGrid.md

```diff
@@ -32,9 +32,16 @@ The following guidance sections describe the additional concepts and techniques
 * [Group, sort and filter data using LINQ and the DataGrid control](datagrid_guidance/group_sort_filter.md)
 * [Editing and input validation in the DataGrid control](datagrid_guidance/editing_inputvalidation.md)
 
-## Sample Project
+## Examples
+
+See the [DataGrid in action](uwpct://Controls?sample=DataGrid) from the [Windows Community Toolkit Sample App](http://aka.ms/uwptoolkitapp).
+
+Here's the [DataGrid sample source code](https://github.com/Microsoft/WindowsCommunityToolkit/tree/master/Microsoft.Toolkit.Uwp.SampleApp/SamplePages/DataGrid).

```

Comment:
The `Sample Project` section is part of our template, so maybe keep it labeled that in the same format?

Reply:
No problem